Corona-Warn-App/src/main/AndroidManifest.xml

```diff
@@ -80,6 +80,7 @@
             android:name=".contactdiary.ui.ContactDiaryActivity"
             android:exported="false"
             android:screenOrientation="portrait"
+            android:launchMode= "singleTop"
```

Comment:
Why is this necessary?

Reply:
Some devices allow to open the contact diary multiple times when clicking fast on the Home Screen card (tested on Huawei P30). SingleTop solves this behaviour